src/generators/go/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './GoGenerator';
+export { GO_DEFAULT_PRESET } from './GoPreset';
+export type { GoPreset } from './GoPreset';
```

Comment:
This is important, as I see in the other parts of PR, some code is still in development, so maybe we should not export at the moment new language? :)